### PR TITLE
Update design-time-code-generation-by-using-t4-text-templates.md

### PR DESCRIPTION
**Pull request by Tim-Maes**
Some inconsistencies fixed and used correct version for VS2022 in code snippet.

### CODE REVIEW

**Comment by Tim-Maes** [comment]
#sign-off

**Comment by Court72** [comment]
@mgoertz-msft

Can you review the proposed changes? 

When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: docs/modeling/design-time-code-generation-by-using-t4-text-templates.md [hunk 1]
```diff
@@ -314,7 +314,7 @@ If you have installed the Visual Studio Modeling SDK, you can have all the templ
 [this blog post](https://devblogs.microsoft.com/devops/the-visual-studio-modeling-sdk-is-now-available-with-visual-studio-2017/).
 
 ```xml
-<Import Project="$(MSBuildExtensionsPath)\Microsoft\VisualStudio\v16.0\TextTemplating\Microsoft.TextTemplating.targets" />
+<Import Project="$(MSBuildExtensionsPath)\Microsoft\VisualStudio\v17.0\TextTemplating\Microsoft.TextTemplating.targets" />
```
Comment (mgoertz-msft) [inline docs/modeling/design-time-code-generation-by-using-t4-text-templates.md:317]: @Court72 Is there a way to make this conditional based on the value selected in the VS version selector for the docs? I.e. v17.0 for VS 2022 and v16.0 for VS 2019.

Comment (Court72) [reply docs/modeling/design-time-code-generation-by-using-t4-text-templates.md:317]: Responded in Teams.

**Review by mgoertz-msft — APPROVED** [review]
#sign-off

**Pull request merged**